src/units/store.rs

```diff
@@ -44,6 +44,22 @@ impl<'a, H: Hasher, D: Data, KB: KeyBox> UnitStore<'a, H, D, KB> {
         self.by_coord.contains_key(coord)
     }
 
+    pub(crate) fn newest_unit(
+        &self,
+        index: NodeIndex,
+    ) -> Option<UncheckedSignedUnit<H, D, KB::Signature>> {
+        let created_units: Vec<_> = self.by_coord.keys().collect();
+        log::trace!(target: "degu", "created_units: {:?}", created_units);
```

Comment:
This log is meant to stay? Or was it for debugging only?

Reply:
Nope, debugging only, got rid of it